assets/common.css

```diff
@@ -403,6 +406,42 @@ thead th {
     width: 1.5rem;
 }
 
+@media (min-width: 1280px) {
+    header nav, main {
+        max-width: 1280px;
+    }
+    #matrix {
+        grid-template-columns: 3fr repeat(17,1fr);
```

Comment:
This line needs to be updated when adding languages. I'm not sure how to avoid that.

Reply:
Many things need to be changed when adding a new language, I think this is a small sacrifice.

An alternative is to use `auto-fit`, specifying a number of columns for the challange header, e.g. 3, rather than resizing the first column. The only thing I don't like about it is that the grid changes size in an unintuitive way when languages are deselected, particularly when many are deselected:

```css
@media (min-width: 1280px) {
  header nav, main {
    max-width: 1280px;
  }
  #matrix {
    grid-template-columns: repeat(auto-fit,minmax(3rem,1fr));
  }
  #matrix a {
    display: initial;
    text-align: center;
  }
  #matrix a svg {
    display: none;
  }
  #matrix .span {
    grid-column: 1/4;
    text-align: right;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
  }
  #matrix input:checked+label {
    grid-column: 4/5;
  }
  #matrix input:checked+label~input:checked+label {
    grid-column: auto;
  }
  #matrix input:not(:checked)+label {
  	order: -1;
    grid-column: 4/5;
  }
  #matrix input:not(:checked)+label~input:not(:checked)+label {
    grid-column: auto;
  }
}
```

Reply:
I agree that it's not a significant problem, I just wanted to point it out since I didn't initially realize it. I like the way it works.

Reply:
I also think it's better as you currently have it, but my opinion is only one. Just wanted to provide an alternative.

Reply:
I tried the alternative. When may languages are deselected and you're trying to deselect more, you have to keep chasing them because they keep moving. 😉

Thanks for providing an alternative option, in case @JRaspass prefers it. It's still better than the current version and I couldn't provide an alternative.